src/Raven.Server/Raven.Server.csproj

```diff
@@ -127,6 +127,8 @@
     <PackageReference Include="Snappier" Version="1.1.6" />
     <PackageReference Include="System.Collections.Immutable" Version="8.0.0" />
     <PackageReference Include="System.Formats.Asn1" Version="8.0.1" />
+    <PackageReference Include="Snowflake.Data" Version="3.1.0"/>
```

Comment:
There is already `4.1.0 ` version, Let's switch to it

Reply:
done